marlowe-playground-client/static/css/modal.scss

```diff
@@ -1,48 +1,93 @@
-/* The Modal (background) */
-.modal {
-  /* Hidden by default */
-  display: block;
+/* TODO: this only works if set on body, need to check how to add it there
+without doing it as an Effect in a mainframe action
+*/
+.avoid-scrolling {
```

Comment:
I don't understand, can't you add it to the `body` css property?

Reply:
I can do it with an effect from handleAction, which is not ideal. One of the things to discuss. 

I was actually hoping for having a css hack that worked on any div but so far no luck

Reply:
@shmish111 do you think is worth adding it as an effect from the mainframe handleAction?

Reply:
I mean why not have in the css:
```scss
body {
  overflow-y: hidden;
}
.avoid-scrolling {
  overflow-y: hidden;
}
```

Reply:
Because if I add that property to boy, you cannot scroll on normal pages. I need to add that whenever I present the modal, and remove it whenever I close it.

To have it always in the body we need the `:has` pseudo class which is still [in draft](https://developer.mozilla.org/en-US/docs/Web/CSS/:has).

With that property we could have 

```css
body:has(.overlay) {
  overflow-y: hidden
}
```

Reply:
ah I see, ok.